metadata.rb

```diff
@@ -9,6 +9,7 @@
 version '0.5.2'
 
 depends 'windows', '~> 1.38'
+depends 'windows_firewall', '~> 3.0.2'
```

Comment:
Can we move this into another PR and include it on the tentacle configure action.  Ideally this needs to be setup when the tentacle is configured not when we want to register